docs/user-guide/103-troubleshooting.md

```diff
@@ -72,7 +72,7 @@ status:
       uuid: 70b556e5-e2da-4e46-a98f-531bc73d0bf5
       type: 802-3-ethernet
       autoconnect: True
-      autoconnect_slaves: <enum NM_SETTING_CONNECTION_AUTOCONNECT_SLAVES_YES of type NM.SettingConnectionAutoconnectSlaves>
+      autoconnect_ports: <enum NM_SETTING_CONNECTION_AUTOCONNECT_SLAVES_YES of type NM.SettingConnectionAutoconnectSlaves>
```

Comment:
Is this really what you see in the traceback? We must be polite while truthful.

Reply:
Right is still autoconnect_slaves